### ISSUE TITLE
Split comment input field into multiple lines on mobile

### ISSUE THREAD

**Issue by NicoHood** [brainstormforce/astra#4924]
### Describe the bug:
This does not look good, the 3 fields next to each other. you can barely enter stuff on your phone.
![image](https://user-images.githubusercontent.com/6888294/212417552-5e18513b-2e10-4e8d-aa59-a810581c4881.png)

### To reproduce:

1. show a post comment form (logged out) on mobile

### Expected behavior:
with this css:
```css
/* Split comment input fields into multiple lines on mobile */
@media (max-width: 768px) {
  .ast-comment-formwrap.ast-row {
    display: inline-block;
  }
  
  #ast-commentform .ast-grid-common-col {
    position: relative;
    width: 100%;
  }
}
```

looks better now:
![image](https://user-images.githubusercontent.com/6888294/212417620-4ae51245-9280-4206-bbc1-67524e77b763.png)

### Isolating the problem:

- [x] This bug happens with no other plugins activated
- [ ] This bug happens with a default WordPress theme active
- [x] I can reproduce this bug consistently using the steps above

### WordPress version:
6.1.1

### Astra Theme version:
4.0.1

**Comment by thebengalboy** [comment]
@NicoHood If I'm not wrong, this is not a bug with Astra. See the default Astra comment section in Mobile- https://d.pr/i/p0LPUT

**Comment by NicoHood** [comment]
Yes, it was fixed with 4.0.2
https://github.com/brainstormforce/astra/releases/tag/v4.0.2
